td/telegram/cli.cpp

```diff
@@ -2546,9 +2546,6 @@ class CliClient final : public Actor {
         status = make_tl_object<td_api::chatMemberStatusCreator>(true);
       } else if (status_str == "uncreator") {
         status = make_tl_object<td_api::chatMemberStatusCreator>(false);
-      } else if (status_str == "admin") {
```

Comment:
I prefer first "admin" case to be deleted. "admin" and "unadmin" was added here nearby together.